src/document/generator.ml

```diff
@@ -1027,10 +1027,10 @@ module Make (Syntax : SYNTAX) = struct
           ++ virtual_ ++ params ++ O.txt " " )
         @ cname @ expr
       in
-      let kind = Some "class-type" in
+      let attr = [ "class-type" ] in
```

Comment:
`["class"; "type"]` ?

Reply:
I don't always have a good grasp on what all these definitions corresponds to visually (In the good old days `odoc` installed the `ocamlary` in `odoc`'s libdir which was nice to develop stylesheets). 

But it's not necessarily smart to split that as you suggest, e.g. if class types are to be treated quite differently from other things labelled `type`.

In any case I didn't review these classifications, but they should certainly be.